<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тестирование сортировок</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="graphic_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>781</width>
        <height>411</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>188</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>кол-во тестов на шаге</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spb_ctests">
              <property name="minimumSize">
               <size>
                <width>91</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>91</width>
                <height>28</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>шаг</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spb_d">
              <property name="minimumSize">
               <size>
                <width>91</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>91</width>
                <height>28</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>макс. кол-во</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spb_count">
              <property name="minimumSize">
               <size>
                <width>91</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>91</width>
                <height>28</height>
               </size>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>198</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="bt_explore">
              <property name="minimumSize">
               <size>
                <width>131</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>131</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Иследовать здесь</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bt_explore_th">
              <property name="minimumSize">
               <size>
                <width>121</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>121</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Исследовать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bt_cancel_th">
              <property name="minimumSize">
               <size>
                <width>81</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>81</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>Отмена</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
